addons/website_sale/static/src/js/website_sale_tour_shop_custom_attribute_value.js

```diff
@@ -24,23 +24,23 @@ odoo.define("website_sale.tour_shop_custom_attribute_value", function (require)
     }, {
         trigger: 'div:contains(Custom: Wood)',
         extra_trigger: '.js_product.in_cart.main_product',
-        run: function (){} // checks that Yep, it's wood!
+        run: function () {} // checks that Yep, it's wood!
```

Comment:
Noise to review, noise to forward-port ;)

Reply:
ok